### ISSUE TITLE
npm repo version issue for 3.0.3 of firebaseui-angular

### ISSUE THREAD

**Issue by choopage** [RaphaelJenni/FirebaseUI-Angular#60]
@RaphaelJenni for version 3.0.3 of https://github.com/RaphaelJenni/FirebaseUI-Angular/blob/master/projects/firebaseui-angular-library/src/lib/firebaseui-angular-library.component.ts 

![screen shot 2018-06-10 at 1 21 53 pm](https://user-images.githubusercontent.com/6467136/41198514-56c9a550-6cb1-11e8-87eb-438215769705.png)
did you see Line 68 of above image
in your /node_modules/firebaseui-angular in (see Line 155 of image below)?

![screen shot 2018-06-10 at 1 19 11 pm](https://user-images.githubusercontent.com/6467136/41198519-6d83a8ae-6cb1-11e8-9a46-492a9977a532.png)

I am currently using the latest version of firebaseui-angular 3.0.3 

![screen shot 2018-06-10 at 1 19 20 pm](https://user-images.githubusercontent.com/6467136/41198528-9b718a10-6cb1-11e8-83c4-48f0c141cfd1.png)

In summary, my version of node_modules/firebaseui-angular is not the same as Github code. May I know if PR code has been pushed to npm package manager? 

I believe it has been pushed because you increased the version from 3.0.2 to 3.0.3. However, I understand that version 3.0.2 and version 3.0.3 are exactly the same code after I installed the library. I am wondering could there be a chance when upgrading to 3.0.3, but the Pull Request of https://github.com/RaphaelJenni/FirebaseUI-Angular/pull/59 did not get push into npm package manager.

I tried many approaches to clean up and verify the package version. i.e. `npm cache clean --force`.

@RaphaelJenni  could you help me verify this? Thank you very much.

**Comment by RaphaelJenni** [comment]
Oh damn, sorry. You are correct. I will correct that immediately.

**Comment by RaphaelJenni** [comment]
3.0.4 is life
